Dockerfile.centos7

```diff
@@ -0,0 +1,177 @@
+# This file was generated! Edits made directly to this file may be lost.
```

Comment:
Probably want to remove this line since I think you made manual modifications to the generated Dockerfile

Reply:
It's actually fully-generated from the templates in the build repo, so I left it to convey that.